cluster-agent/values.yaml

```diff
@@ -3,14 +3,22 @@ deployment:
   image:
     repository: docker.io/appdynamics/cluster-agent
     tag: latest
-    pullPolicy: IfNotPresent
+    pullPolicy: Always
   resources:
     limits:
       cpu: 200m
-      memory: "300M"
-    requests: 
+      memory: "300Mi"
+    requests:
       cpu: 100m
-      memory: "100M"
+      memory: "100Mi"
+
+resources:
+  limits:
+    cpu:
+    memory:
+  requests:
+    cpu:
+    memory:
```

Comment:
Adding resources in values.yaml as they are used by the chart.
Values that are set in values.yaml appear as configurable options while deploying chart using ICP UI.
On another note: Do we need deployment.resources as well resources? Will it be okay to keep only one config option?